2024/wg-fedid.html

```diff
@@ -167,17 +167,26 @@ <h2>Motivation and Background</h2>
     <section id="scope" class="scope">
       <h2>Scope</h2>
       <p>
-        The Working Group will specify new web platform features intended to be implemented in user agents like browsers. The purpose of these features is to support privacy-preserving authentication, authorization flows, and requesting federated identities without compromising security principles for Identity Providers (IdPs) or Relying Parties (RPs) (in a ‘traditional’ federation model) or Issuers, Verifiers, and Holders (in a digital identity wallet architecture), and User Agents. Here, “privacy” minimally refers to the appropriate processing of personal information and preventing third parties from gleaming anything about the end-user’s environment (e.g., which wallets are available and their capabilities). This work results in developing new mechanisms that define how information is passed by the browser between the different entities and authentication intermediaries to facilitate federated authentication; these mechanisms are not authentication methods.
+        The Federated Identity Working Group defines Web Platform features that allow user agents to support secure and privacy-preserving interactions related to digital identities. 
```

Comment:
```suggestion
        The Federated Identity Working Group defines Web Platform features that enable user agents to support secure and privacy-preserving interactions related to digital identities. 
```

Reply:
"related to digital identities" -> related to digital credentials